### DISCUSSION TITLE
CIM-CGMES format

### DISCUSSION THREAD

**Discussion by olav-lafourcade** [PowerGridModel/.github#26] [Q&A] [answered]
Hello,
In the presentation you gave on the 01/06/23 for the LF Energy (https://youtu.be/lkNj2EzxelA?si=bf5xfmaD61wccgke) you mention that PGM supports CIM-CGMES format and that Alliander uses it as their backend format for grid modeling. However, I can't find resources on how to load and convert a CIM-CGMES format to PGM model. Do you have any ressources/doc/insight on this topic?
Sincerely,
Olav

**Comment by nitbharambe** [comment, marked as answer]
Hi @olav-lafourcade ,
You can have a look here: https://github.com/PowerGridModel/IEC-CIM-conversion-services-beta

**Reply by petersalemink95** [reply]
Hi @olav-lafourcade,

To give you a bit more detail:
Within Alliander we currently use CIM data and we internally have a converter for it. 
This converter is not open-sourced.
However, it is on our roadmap to open-source a CIM-CGMES converter in the future. 

RSE Italy made a converter for their internal usage; the code is present in the link that @nitbharambe sent you.
However, this is not a supported package yet. 
I hope this already gives you the input you needed.
If not, know that we will work on a CIM-CGMES converter in the future.

**Reply by olav-lafourcade** [reply]
Hi @petersalemink95 , @nitbharambe 
Thank you for both answers, this is very helpful !
